starcheck/src/aca_load_review_cl.rst

```diff
@@ -330,7 +330,7 @@ Checks
 +-------+------------------+-+-+-------------------------------------------+----+----------------+
 |       |                  |X|X|                                           |    |AS: Possible    |
 |       |                  | | |                                           |    |Bright Star     |
-|ACA-041|Magnitude         | | |Slot MAXMAG (faint limit) - star MAG >= 1.4|n/a |Hold, GS:       |
+|ACA-041|Magnitude         | | |Slot MAXMAG (faint limit) >= star MAG      |n/a |Hold, GS:       |
```

Comment:
Thoughts?

Reply:
OK.  Per our discussion I'm going to chuck this and go  MAXMAG is more than MAG + 0.5 or MAXMAG == 11.2 .  

MAXMAG >= min(MAG + 0.5, 11.2)

Reply:
I suppose in our current operational posture it should also be 11.2 >= MAXMAG >= min(MAG + 0.5, 11.2), but I don't think we care about adding a check that MAXMAG is not greater than 11.2

Reply:
What about just changing 1.4 to 0.3.  So basically the checklist item should never change again and it is a simple, hard limit.  Starcheck is free to be more subtle.  Then add a second condition MAXMAG <= 11.2.  That is another hard limit that we are very unlikely to change.

Reply:
Did you mean 0.2 hard limit? (putting this comment back in the thread).  I had read 11.0 as the non-temperature scaled ACQ star faint limit from here:

https://github.com/sot/proseco/blob/ad6f861d0fd9eb27cc7034731ed758c9b875f84b/proseco/acq.py#L285

And by the time we got to maxmag - mag >= 0.2, I had just proposed changing the checklist item to just having no margin (maxmag >= mag) which is how this PR started.  But then I was sanity checking in starcheck to see that we were still getting reasonable values.

Reply:
With the 11.2 maxmag we could consider pushing down the acq selection limit.  The reason that limit is not temperature scaled is that there is an accurate temperature-dependent model of star acq performance, so the upper bar can just be set at an arbitrary high but not ridiculously high limit.    We could drop it down to 10.9 without any impact, except for wasting time re-validating.  Or we could just put a 0.3 value in the checklist and never notice, in practice.  I think that we would not ever want less than 0.3 mag margin.

Reply:
I'm fine with updating the checklist to use 0.3 mag margin.  If we go with that I think it would also make sense to update the "risks" section, to indicate that less than that could reduce acq probability or ability to track that single star.  ACA-041 looks scarier than it needs to be with "bright star hold" and "reduced aspect quality".